bundles/org.openhab.binding.intesis/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="intesis"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="intesisHome">
+		<label>IntesisHome WiFi Adapter</label>
+		<description>IntesisHome WiFi Adapter</description>
+
+		<channels>
+			<channel id="power" typeId="system.power"/>
+			<channel id="mode" typeId="mode"/>
+			<channel id="windspeed" typeId="windspeed"/>
+			<channel id="swingUpDown" typeId="swingUpDown"/>
+			<channel id="temperature" typeId="temperature"/>
+			<channel id="returnTemp" typeId="returnTemp"/>
+			<channel id="outdoorTemp" typeId="outdoorTemp"/>
+		</channels>
+
+		<config-description>
+			<parameter name="password" type="text" required="true">
+				<label>Password</label>
+				<description>Password for API access.</description>
+				<context>password</context>
+			</parameter>
+			<parameter name="ipAddress" type="text" required="true">
+				<label>IP Address</label>
+				<description>IP Address of the IntesisHome device.</description>
+				<context>network-address</context>
+			</parameter>
```

Comment:
I think it's more common if the IP address is entered first and after that, the password.

Reply:
Changed